migrations/20141229012503_00.js

```diff
@@ -9,6 +9,7 @@ exports.up = function(knex, Promise) {
     table.string('hn_url');
     table.string('gh_url').primary();
     table.string('gh_name');
+    table.integer('gh_stars');
```

Comment:
We can't edit the existing migrations since they're already deployed :(
Please make a new migration.